Jasmine/Grid/SquareTextView.swift

```diff
@@ -0,0 +1,68 @@
+import UIKit
+import ChameleonFramework
+
+/// Displays a view that is square in shape.
+/// Capable of displaying a limited number of character(s), including chinese characters.
+/// 
+/// - Author: Wang Xien Dong
+@IBDesignable
+class SquareTextView: UILabel {
+
+    /* Constants */
+    private static let heightWidthRatio = CGFloat(1.0)
+    private static let borderWidth = CGFloat(2.0)
+    private static let filledColor = FlatMint()
+    private static let emptyColor = UIColor.clear
```

Comment:
If the constants have general reuse potential, then move them there.

I'd move `FlatMint()`. `emptyColor` feels a little redundant.